packages/ag-charts-community/src/chart/toolbar/toolbarStyles.css

```diff
@@ -135,6 +135,10 @@
     white-space: nowrap;
 }
 
+.ag-charts-toolbar__button[data-toolbar-group='ranges'] {
```

Comment:
Seems like a good way to target this. What do you think of moving this to the bottom and have a section like `specific overrides`, so it's clear where we're doing this sort of stuff